renderer/utils/reports.ts

```diff
@@ -278,6 +278,9 @@ export function checkIntersection(previousTo: string, currentFrom: string) {
 export function validation(activities: Array<ReportActivity>) {
   try {
     for (let i = 0; i < activities.length; i++) {
+      const [toHours, toMinutes] = activities[i].to.split(":");
+      const [fromHours, fromMinutes] = activities[i].from.split(":");
```

Comment:
you can do something like: const [toHours, toMinutes] = activities[i].to.split(":").map((item) => Number(item))
to get numbers and avoid write Number in 306-313 lines